aiocache/backends/valkey.py

```diff
@@ -147,15 +188,17 @@ async def _raw(self, command, *args, encoding="utf-8", _conn=None, **kwargs):
         return value
```

Comment:
The line above (can't comment on it because it's not in the diff) is no longer covered (command is never keys or mget).